Pipfile

```diff
@@ -17,6 +16,7 @@ pytest-clarity = "*"
 autopep8 = "*"
 gspread =  "*"
 oauth2client = "*"
+pylint = "*"
```

Comment:
Quick question here, does the order here matter at all?

Reply:
Nope